common/raw_hashtable.h

```diff
@@ -1524,6 +1644,19 @@ auto TableImpl<InputBaseT, SmallSize>::small_storage() const -> Storage* {
   }
 }
 
+// Helper to set up the storage of a table when a specific size has already been
+// set up. If possible, uses any small storage, otherwise allocates.
+template <typename InputBaseT, ssize_t SmallSize>
+auto TableImpl<InputBaseT, SmallSize>::SetUpStorage() -> void {
+  CARBON_DCHECK(this->small_alloc_size() == SmallSize);
+  ssize_t local_size = this->alloc_size();
+  if (SmallSize > 0 && local_size == SmallSize) {
```

Comment:
```suggestion
  CARBON_DCHECK(this->small_alloc_size() == SmallSize);
  ssize_t local_size = this->alloc_size();
  CARBON_DCHECK(local_size != 0);
  if (local_size == SmallSize) {
```
I think we can't have `SmallSize == 0 && local_size == SmallSize` here, because we should never call this when `local_size == 0`.

Reply:
Ah, true, thanks!